src/Core/tests/UnitTests/Layouts/GridLayoutManagerMinMaxTests.cs

```diff
@@ -0,0 +1,808 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+using NSubstitute;
+using Xunit;
+using static Microsoft.Maui.UnitTests.Layouts.LayoutTestHelpers;
+using static Microsoft.Maui.UnitTests.Layouts.GridLayoutManagerTests;
+
+namespace Microsoft.Maui.UnitTests.Layouts
+{
+    [Category(TestCategory.Layout)]
+    public class GridLayoutManagerMinMaxTests
```

Comment:
Can you actually refactor the existing code in src/Core/tests/UnitTests/Layouts/GridLayoutManagerTests.cs to be a partial class and this one can just add new mthods.

Name this file src/Core/tests/UnitTests/Layouts/GridLayoutManagerTests.MinMax.cs